docs/_extra/api-reference/schemas/organization.yaml

```diff
@@ -6,5 +6,9 @@ Organization:
   properties:
     id:
       type: string
+    logo:
+      type: string
+      format: uri
```

Comment:
I think we can probably just say that this will always be a URL, not just a uri.

Reply:
Sure, but I don't think `url` is a supported `format` in JSON Schema (where the Open API/Swagger formats "come from"):

![image](https://user-images.githubusercontent.com/439947/38424473-29983dfe-397f-11e8-8d69-cfdbc4226b5a.png)